apps/AEPSampleAppNewArchEnabled/README.md

```diff
@@ -45,10 +45,9 @@ yarn run build
 
 ## Get Started
 
-### Configure the App ID
+### Add your App Id:
 
-- **iOS**: In `apps/AEPSampleAppNewArchEnabled/ios/AEPSampleAppNewArchEnabled/AdobeBridge.m`, find the `configureWithAppId` call and add your app ID.
-- **Android**: In `apps/AEPSampleAppNewArchEnabled/android/app/src/main/java/com/AEPSampleAppNewArchEnabled/MainApplication.kt`, find the `configureWithAppId` call and add your app ID.
+In `apps/AEPSampleAppNewArchEnabled/app/layout.tsx`, locate the call to `MobileCore.initializeWithAppId(appId:"YOUR_APP_ID")` and replace `"YOUR_APP_ID"` with your property App Id.
```

Comment:
Should we follow the file name `_layout.tsx`?
apps/AEPSampleAppNewArchEnabled/app/_layout.tsx